collections/_posts/2022-11-26-SparseMultiChannelVAE.md

```diff
@@ -0,0 +1,311 @@
+
+It is beneficial to use continuous noise instead of discrete one because **multiplying the inputs by a Gaussian noise is equivalent to putting Gaussian noise on the weights**.
+This procedure can be used to obtain a posterior distribution over the model’s weights ([Wang & Manning. 2013](http://proceedings.mlr.press/v28/wang13a.pdf) ; [Kingma et al., 2015](https://proceedings.neurips.cc/paper/2015/file/bc7316929fe1545bf0b98d114ee3ecb8-Paper.pdf)).
+
+
+The elements $$x_i$$ are approximately Gaussian for the Lyapunov’s central limit theorem and their distributions have the form:
+
+$$x_i \sim  \mathcal{N} (\sum_{k}^{} \theta_{ik}; \alpha \sum_{k}^{} \theta^2_{ik} )$$ 
+
+with $$\alpha = p/1−p$$ and $$ \theta_{ik} = g_{ik}z_{k}(1 − p) $$.
+
+
+## Variational dropout and sparsity
+
+Posterior distributions on the encoder weights $$w$$ that take the form $$w \sim  \mathcal{N} (\mu; \alpha \mu^2)$$ are called dropout posteriors.
+
+**If the variational posteriors on the encoder weights are dropout posteriors, Gaussian dropout arises.**
+
+The improper log-scale uniform **is the only prior distribution that makes variational inference consistent with Gaussian Dropout** ([Kingma et al., 2015](https://proceedings.neurips.cc/paper/2015/file/bc7316929fe1545bf0b98d114ee3ecb8-Paper.pdf)) :
+
+$$p (ln |w|) = const \Leftrightarrow  p (|w|) \propto  \frac{1}{|w|} $$
+
+
+
+With this prior, the $$D_{KL}$$ of the dropout posterior depends only on $$\alpha$$ and can be numerically approximated ([Molchanov et al., 2017](http://proceedings.mlr.press/v70/molchanov17a/molchanov17a.pdf)):
+
+$$D_{KL} (\mathcal{N} (w; \alpha w^2) || p (w) ) ≈ −k_1\sigma(k_2 + k_3 ln \; \alpha) + 0.5 ln(1 + \alpha^{−1}) + k_1 $$
+
+$$k_1 = 0.63576$$, $$k_2 = 1.87320$$, $$k_3 = 1.48695$$, $$\sigma$$(·) Sigmoid function.
+
+The optimization of $$D_{KL}$$ promotes $$\alpha \to \infty $$ ($$\alpha = p/1−p$$), the implicit drop rate $$p$$ tends to 1. The associated weight $$w$$ can be discarded. 
+
+Unless that weight is beneficial for the optimization objective, that is to maximize the data log-likelihood, it will be set to zero.
+**Sparsity arises naturally**.
+
+
+
+## Sparse Multi-Channel VAE
+
+This part will be completed later (After the post on (Sparse) Variational Dropout).
+
+
+
+&nbsp;
+
+# Results
+
+## Medical imaging data
+
+The dataset is composed of
+
+* 504 subjects of the Alzheimer's Disease Neuroimaging Initiative (ADNI) database
+* Clinical channel is complsed by six continuous variables: age, results to mini-mental state examination, adas-cog, cdr, faq tests, scholarship level
+* Three imaging channels: structural MRI, functional FDG-PET, Amyloid-PET. For each modality, 90 image intensities were computed from 90 brain regions mapped in the AAL atlas. This strategy produces 90 features arrays for each image. Lastly, data was centered and standardized across features. 
```

Comment:
The current phrasing makes it a bit ambiguous if 90 intensities are computed for **each** region. So I propose to clarify this:
```suggestion
* Three imaging channels: structural MRI, functional FDG-PET, Amyloid-PET. For each modality, the average image intensity was computed across 90 brain regions mapped in the AAL atlas. This strategy produces 90 features arrays for each image. Lastly, data was centered and standardized across features. 
```